dobble/__main__.py

```diff
@@ -45,11 +45,12 @@ def main(symbols_folder: str,
     """
```

Comment:
@dscho 
One of the tricky points is to make sure that the position, rotation and scale that have been optimized for the low-resolution rasterized mask is then correctly applied on the original input SVG, as currently done on the input  high-resolution image

Reply:
not necessarily hard to do, but crucial to be done perfectly